ykman/piv.py

```diff
@@ -256,6 +256,25 @@ class SW(IntEnum):
     AUTHENTICATION_BLOCKED = 0x6983
     INCORRECT_PARAMETERS = 0x6a80
 
+    @classmethod
+    def is_verify_fail(cls, sw):
+        return 0x63c0 <= sw <= 0x63c3
```

Comment:
It's not quite clear to me why we're just considering values 0-3 as valid here, but the `tries_left` function handles a much larger span.

Reply:
Yeah, I agree. The APDU list says

High byte | Low byte | Severity | Explanation
-- | -- | -- | --
63 | C0 | W | Verify fail, no try left.
63 | C1 | W | Verify fail, 1 try left.
63 | C2 | W | Verify fail, 2 tries left.
63 | C3 | W | Verify fail, 3 tries left.
63 | CX | W | The counter has reached the value 'x' (0 = x = 15) (command dependent).

so maybe we should expand `is_verify_fail`? I'm not sure which is more correct.

In actuality, the reason for the discrepancy is that `tries_left` was moved here from `PivController._parse_tries_left` and `is_verify_fail` was newly written before we decided to move `_parse_tries_left`.

Reply:
Bump

Reply:
I think 63CX should be seen as a "Verify fail" as well, no?

Reply:
Agreed

Reply:
All right, will do!